packages/amplify-appsync-simulator/src/server/subscription.ts

```diff
@@ -5,7 +5,8 @@ import { createServer as createHTTPServer } from 'http';
 import * as e2p from 'event-to-promise';
 import * as portfinder from 'portfinder';
 
-import { Server as CoreHTTPServer, AddressInfo } from 'net';
+import { Server as CoreHTTPServer } from 'net';
+import { AddressInfo } from 'dgram';
```

Comment:
Why? Is this because of updated typescript/nodejs?

Reply:
Because Node version was changed back to the lower version what we support and it has the AddressInfo elsewhere.